scalafix-core/src/main/scala/scalafix/util/TokenClasses.scala

```diff
@@ -3,6 +3,7 @@ package scalafix.util
 import scala.meta.classifiers.Classifier
 import scala.meta.tokens.Token
 import scala.meta.tokens.Token._
+import meta.classifiers.XtensionClassifiable
```

Comment:
is meta the root package or is that a relative import?

Reply:
it's a relative import, will fix all of them